test/e2e/network/dns_configmap.go

```diff
@@ -220,14 +221,15 @@ func (t *dnsPtrFwdTest) run() {
 
 var _ = SIGDescribe("DNS configMap nameserver", func() {
 
+	nsTest := &dnsNameserverTest{dnsTestCommon: newDnsTestCommon()}
+	fwdTest := &dnsPtrFwdTest{dnsTestCommon: newDnsTestCommon()}
```

Comment:
Interesting, didn't notice `newDnsTestCommon()` is embedded with `NewDefaultFramework()`, which is embedded with `BeforeEach` and `AfterEach`.

By calling `newDnsTestCommon()` twice, we push two BeforeEach/AfterEach hooks, which also seems improper. Can we move `framework.NewDefaultFramework()` out of `newDnsTestCommon` so we don't call it twice?

Reply:
Wrap two tests into different contexts. This should do the trick. (*msg will be squashed